content/rs/databases/redis-on-flash/_index.md

```diff
@@ -16,138 +16,101 @@ aliases: [
-Redis on Flash can cut resource costs by over 70% when compared to an
-all-RAM Redis Enterprise Software deployment.
+standard Redis databases must all be in RAM, Redis on Flash enables your databases to span both RAM and dedicated flash memory in the form of SSDs (solid state drives).
+
+Distributing data between RAM and flash memory keeps your frequently used data accessible in RAM, and values you need less often in flash memory. Since flash memory usually comes with more storage space and less cost than RAM, this can result in significant savings for large datasets.
+
+Just like all-RAM databases, RoF is compatible with existing Redis applications. Databases that employ RoF are identical to all-RAM Redis Enterprise Software databases in characteristics and features.
+
+Redis on Flash is also supported on [Redis Cloud]({{<relref "/rc/">}}) and [Redis Enterprise Software for Kubernetes]({{<relref "/rs/">}}).
 
 ## Flash memory
 
-Unlike standard Redis Enterprise Software installations, implementing
-Redis on Flash requires pre-planning around memory and overall sizing.
-There are a few critical recommendations:
-
-- The flash memory should be local to the server/VM/instance/container
-    as opposed to network attached.
-- The flash memory should be dedicated to RoF and not shared with other
-    parts of the database, (e.g. durability, binaries, etc.).
-- The flash memory should be [NVMe
-    based](https://en.wikipedia.org/wiki/NVM_Express) for best
-    performance.
-
-For more information read [Ephemeral and Persistent Storage in Redis
-Enterprise
-Software]({{< relref "/rs/installing-upgrading/persistent-ephemeral-storage.md" >}}).
-
-When running on a cloud environment, the flash memory for Redis on Flash
-should be on the ephemeral SSDs of the cloud instance and persistent
-database storage should be network attached, e.g AWS EBS. For AWS, we
-specifically recommend "[Storage Optimized I3 - High I/O
-Instances](https://aws.amazon.com/ec2/instance-types/#storage-optimized)"
-because of the performance of NVMe for flash memory.
-
-When deploying RoF on an on-premises environment, local SSD (preferably NVMe-based) should be used. 
-To be clear, running RoF over Network Attached Storage (NAS), Storage Area Network (SAN), or with local HDD drives isn’t supported.  
- 
-Please note - the Redis Enterprise Software database persistent and
-ephemeral storage should be on different disks, either local or attached.
+Implementing Redis on Flash requires preplanning around memory and sizing. Considerations and requirements for Redis on Flash include:
+
+- Flash memory must be locally attached (as opposed to network attached)
+- Flash memory must be dedicated to RoF and not shared with other parts of the database (e.g. durability, binaries, persistence, ect.).
+- For the best performance, the SSDs should be NVMe based.
+
+Flash memory can be SATA or NVMe based storage devices. However, we recommend NVMe for the best performance.
```

Comment:
I think we don't need both of those lines since they seem to say the same thing.